openpathsampling/storage/stores/snapshot_wrapper.py

```diff
@@ -143,7 +142,7 @@ def load(self, idx):
                     return self.storage.fallback.stores[self.name].load(idx)
                 else:
                     raise ValueError(
-                        'str %s not found in storage or fallback' % idx)
+                        'long %s not found in storage or fallback' % idx)
```

Comment:
"long" of just "int" (or maybe "object ID")? Thinking that Py3 doesn't distinguish `long`

Reply:
changed